common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1930,6 +1931,15 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("1m")
 
+  val WORKER_PARTITION_SPLIT_MAX_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.shuffle.partitionSplit.max")
+      .withAlternative("celeborn.shuffle.partitionSplit.max")
+      .categories("worker")
+      .doc("Specify the maximum partition size for splitting, and ensure that individual partition files are always smaller than this limit.")
+      .version("0.3.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("16g")
```

Comment:
is this too big? I know your case the data is extremely huge, but the default value should be based on the common cases.

Reply:
On our production, 4GB is a set value, but it is only used to limit the file size in worst-case scenarios, so a very large value is given. In typical scenarios, this value should not be set.

Reply:
Given the default value of `celeborn.client.shuffle.partitionSplit.threshold` is 1G, in the normal case, the real split partition size should be a little larger than 1G, set it to 2G as an HRAD_SPLIT threshold should be fine. Otherwise, users may be surprised if they see an extremely large file than they wanted.

Reply:
Okay, I'm done with the changes